VideoWeb/VideoWeb/ClientApp/src/app/app.component.ts

```diff
@@ -90,7 +90,11 @@ export class AppComponent implements OnInit, OnDestroy {
     private async attemptRetrieveProfile(loggedIn: boolean) {
         if (!loggedIn) {
             const currentUrl = this.locationService.getCurrentUrl();
-            this.router.navigate([`/${pageUrls.IdpSelection}`], { queryParams: { returnUrl: currentUrl } });
+            if (currentUrl.indexOf(pageUrls.Logout) > -1) {
+                this.router.navigate([`/${pageUrls.Logout}`]);
```

Comment:
hey @ZJianWang - can you explain this change, pls?

Reply:
looks this change need to revert I saw Nick already had a fixed for logout issue